cloud/functions/src/crawlers/devoxx/crawler.ts

```diff
@@ -0,0 +1,186 @@
+import {info} from "../../firebase";
+import * as _ from "lodash";
+
+import { CfpEvent, DevoxxScheduleItem, DevoxxScheduleSpeakerInfo } from "./types"
+import { DailySchedule, Speaker, Talk } from "../../../../../shared/dayly-schedule.firestore"
+import { TalkStats } from "../../../../../shared/feedbacks.firestore";
+import { FullEvent } from "../../models/Event";
+import { ISODatetime, ISOLocalDate } from "../../../../../shared/type-utils";
+import { Day, ListableEvent } from "../../../../../shared/event-list.firestore";
+import { Temporal } from "@js-temporal/polyfill";
+
+const axios = require('axios');
+
+const daysOfWeek = ['monday', 'tuesday', 'wednesday', 'thursday', 'friday', 'saturday', 'sunday']
+
+export const crawl = async (eventId:string) => {
+    const res = await axios.get(`https://${eventId}.cfp.dev/api/public/event`)
+    const e: CfpEvent = res.data;
+
+    const start = e.fromDate.substring(0, 10) as ISOLocalDate
+    const end = e.toDate.substring(0, 10) as ISOLocalDate
+
+    // collect days
+    const days: Day[] = []    
+    for(let d:Temporal.PlainDate = Temporal.PlainDate.from(start); ; d = d.add({days: 1})) {
+        days.push({id: daysOfWeek[d.dayOfWeek - 1], localDate: d.toString() as ISOLocalDate})
+        if (d.toString() == end) {
+            break;
+        }
+    }    
+
+    const eventInfo = {
+        id: eventId,
+        title: e.name,
+        description: e.description,
+        peopleDescription: "",
+        timezone: e.timezone,
+        start: start,
+        end: end,
+        days: days,
+        logoUrl: guessLogoUrl(e),
+        backgroundUrl: e.eventImageURL,
+        websiteUrl: e.website,
+        location: { city: e.locationCity, country: e.locationCountry },
+        theming: {
+            colors: {
+                primaryHex: "#F78327",
+                primaryContrastHex: "#FFFFFF",
+                secondaryHex: "#3880FF",
+                secondaryContrastHex: "#FFFFFF",
+                tertiaryHex: "#0F0F0F",
+                tertiaryContrastHex: "#FFFFFF"
+            }
+        },
+        keywords: [ "Devoxx", "Java", "Kotlin", "Cloud", "Big data", "Web" ]
+      } as ListableEvent
+
+    const event: FullEvent = { id: eventId, info: eventInfo, daySchedules: [], talkStats: [], talks: []}
+    for (const day of days) {
+        const {daySchedule, talkStats, talks} = await crawlDevoxxDay(eventId, day.id)
+        event.daySchedules.push(daySchedule)        
+        event.talkStats.push({day: day.id, stats: talkStats})
+        for (const talk of talks) {
+            event.talks.push(talk)
+        }
+    }
```

Comment:
```suggestion
    await Promise.all(days.map(async day => {
        const {daySchedule, talkStats, talks} = await crawlDevoxxDay(eventId, day.id)
        event.daySchedules.push(daySchedule)
        event.talkStats.push({day: day.id, stats: talkStats})
        for (const talk of talks) {
            event.talks.push(talk)
        }
    }))
```
Let's parallelize days : went from 19s to 17s by doing this locally